.github/workflows/release-chart.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         id: get_version
         run: echo "::set-output name=version::${GITHUB_REF_NAME#helm/}"
       - name: Push chart to GitHub Container Registry
-        uses: appany/helm-oci-chart-releaser@v1.0.0
+        uses: sharekey/helm-oci-chart-releaser@v1.0.0
```

Comment:
I'm not convinced by this. That repo differs only for that one commit, but it's a 0 stars fork of the original (which in turn is not really popular to start with).

I'd stay with the original one and maybe send the patch upstream if we need it. Or fork it altogether, since it's a quite high profile component with release privileges.

wdyt?